### PR TITLE
curvefs: fix refresh inode in getInodeAttr everytime when read

### PR DESCRIPTION
**Pull request by SeanHai**
Signed-off-by: wanghai01 <seanhaizi@163.com>

### What problem does this PR solve?

Issue Number: #xxx 
#1855 

Problem Summary:

### What is changed and how it works?

What's Changed:

How it Works:

Side effects(Breaking backward compatibility? Performance regression?):

### Check List

- [ ] Relevant documentation/comments is changed or added
- [ ] I acknowledge that all my contributions will be made under the project's license

### CODE REVIEW

File: curvefs/src/client/curve_fuse_op.cpp [hunk 1]
```diff
@@ -417,6 +417,8 @@ void FuseOpOpen(fuse_req_t req, fuse_ino_t ino, struct fuse_file_info *fi) {
         FuseReplyErrByErrCode(req, ret);
         return;
     }
+    // set fh to 1 to indicate needn't refresh inode when file is still opened
+    fi->fh = 1;
```
Comment (cw123) [inline curvefs/src/client/curve_fuse_op.cpp:421]: don't use magic word

File: curvefs/src/client/fuse_client.cpp [hunk 2]
```diff
@@ -964,11 +964,13 @@ CURVEFS_ERROR FuseClient::FuseOpGetAttr(fuse_req_t req, fuse_ino_t ino,
                                         struct stat *attr) {
     VLOG(1) << "FuseOpGetAttr ino = " << ino;
     if (FLAGS_enableCto) {
-        CURVEFS_ERROR ret = inodeManager_->RefreshInode(ino);
-        if (ret != CURVEFS_ERROR::OK) {
-            LOG(ERROR) << "inodeManager get inode fail, ret = " << ret
-                       << ", inodeid = " << ino;
-            return ret;
+        if (fi == nullptr || fi->fh != 1) {
```
Comment (ilixiaocui) [inline curvefs/src/client/fuse_client.cpp:967]: what's the initialize value of fi->fh?

File: curvefs/src/client/curve_fuse_op.cpp (hunk 1 above)
Comment (SeanHai) [reply curvefs/src/client/curve_fuse_op.cpp]: fixed

File: curvefs/src/client/fuse_client.cpp (hunk 2 above)
Comment (SeanHai) [reply curvefs/src/client/fuse_client.cpp]: 0 the default value of uint64_t and have changed to enum value which is more meaningful.

**Comment by SeanHai** [comment]
recheck

**Comment by SeanHai** [comment]
recheck

**Review by ilixiaocui — APPROVED** [review]

**Review by cw123 — APPROVED** [review]

**Pull request merged**